node/network/collator-protocol/src/collator_side.rs

```diff
@@ -1085,25 +1072,40 @@ mod tests {
 			}
 		);
 
+		let current_index = 1;
```

Comment:
A comment here would be nice, explaining why the current index is expected to be 1.